api-server.py

```diff
@@ -101,9 +118,37 @@ def main():
     elif args.coznet:
         settings.setup_coznet()
 
+    syslog_facility = None
+    if args.syslog or args.syslog_local:
+        # Setup the syslog facility
+        if args.syslog_local:
+            print("Logging to syslog local facility %s" % args.syslog_local)
```

Comment:
I'd suggest changing this to:

```
print("Logging to syslog local%s facility" % args.syslog_local)
```
for clarity.

Reply:
👍